argusclient/client.py

```diff
@@ -139,17 +139,19 @@ def __init__(self, argus):
 
 
 class BaseModelServiceClient(object):
-    def __init__(self, argus, get_all_path=None):
+    def __init__(self, argus, get_all_path=None, get_all_params=None):
```

Comment:
@hdara-sfdc Were the changes in this file fine? Wondering if the design/implementation is fine so that I can do the same for Permissions and Dashboards Service Clients

Reply:
You mean, the support for `get_all_params`? Yes, it should be fine.

Reply:
Great, thanks